src/data-source.handlers.ts

```diff
@@ -19,51 +19,51 @@ interface ResourceHandlerMap {
 export default {
   create: (req, res, next): void => {
     const data = omit(req.body, 'id');
-    const { allowedOrganizations } = res.locals;
+    const { allowedOrganizations, isSysAdmin } = res.locals;
 
     if (
-      allowedOrganizations &&
-      allowedOrganizations.length > 0 &&
-      !allowedOrganizations.includes(data.publisherId)
+      isSysAdmin ||
```

Comment:
Med "optional chaining" skal den vel kunne forkortes til:
isSysadmin || allowedOrganizations?.includes(data.publisherId)

Reply:
1bcaf94 :+1:

Reply:
Det feila visst...

Reply:
Kjører den for gammel versjon av node?

Reply:
Ser ut til å gå bra når jeg oppdaterte til 14: 8ac61ca